src/NUnitFramework/tests/Syntax/AfterTests.cs

```diff
@@ -55,7 +55,7 @@ public class AfterTest_AndOperator : SyntaxTest
         [SetUp]
         public void SetUp()
         {
-            parseTree = "<after 1000 <and <greaterthan 0> <lessthan 10>>>";
+            parseTree = "<after 1000 millisecond <and <greaterthan 0> <lessthan 10>>>";
```

Comment:
It makes sense to add more tests to covert different intervals here. Will do that soon.